packages/patternfly-3/react-console/src/VncConsole/VncActions.js

```diff
@@ -1,30 +1,60 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import { MenuItem, Button, helpers } from 'patternfly-react';
 
 const { Dropdown } = Button;
 const { noop } = helpers;
 
-const VncActions = ({ textSendShortcut, textCtrlAltDel, onCtrlAltDel }) => (
-  <Dropdown bsStyle="default" title={textSendShortcut} id="console-send-shortcut" onClick={noop}>
-    <MenuItem eventKey="1" onClick={onCtrlAltDel}>
-      {textCtrlAltDel}
-    </MenuItem>
-  </Dropdown>
-);
+const VncActions = ({
+  textSendShortcut,
+  textCtrlAltDel,
+  textDisconnect,
+  onCtrlAltDel,
+  onDisconnect,
+  portalToolbarTo
```

Comment:
The name is too specific for the `oVirt VM Portal`

Reply:
Well, it calling `portalToolbarTo`, because if it set it do portal toolbar to some element, not because this is from VM Portal. But yeah, it better to rename.